doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,302 @@
+.. _ncs_release_notes_100:
+
+|NCS| v1.0.0 Release Notes
+##########################
+
+This project is hosted by Nordic Semiconductor to demonstrate the integration of Nordic SoC support in open source projects, like MCUBoot and the Zephyr RTOS, with libraries and source code for low-power wireless applications.
+
+nRF Connect SDK v1.0.0 supports development with nRF9160 Cellular IoT devices.
+It contains references and code for Bluetooth Low Energy devices in the nRF52 Series, though development on these devices is not currently supported with the nRF Connect SDK.
+
+Highlights
+**********
+
+*
+*
+*
+*
+*
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.0.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK) - only for LTE samples
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+* PCA63519 (Smart Remote 3 DK add-on)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.41
+     - `J-Link Software and Documentation Pack`_
+   * - nRF5x Command Line Tools
+     - v9.8.1
+     - `nRF5x Command Line Tools`_
+   * - dtc (Linux only)
+     - v1.4.6 or later
+     - :ref:`gs_installing_tools_linux`
+
+
+As IDE, we recommend to use |SES| (Nordic Edition), version 4.16 or later.
+It is available from the following links:
+
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+* `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+
+Change log
+**********
+
+The following sections provide detailed lists of changes by component.
+
+nRF9160
+=======
+
+* Added the following samples:
+
+  * :ref:`aws_fota_sample` - shows how to perform over-the-air firmware updates of an nRF9160 via MQTT and HTTP.
+  * :ref:`secure_partition_manager` - required to set up the nRF9160 DK so that it can run user applications in the non-secure domain.
```

Comment:
Remove this as this sample was added in 0.4

Reply:
done